modules/dynamicfusion/include/opencv2/io/capture.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include <opencv2/kfusion/kinfu.hpp>
+#include <string>
+
+namespace cv
+{
+    namespace kfusion
+    {
+        class KF_EXPORTS OpenNISource
```

Comment:
Looks like this class is not used anymore, did you try to use `VideoCapture` with OPENNI backend?
```
VideoCapture cap(CAP_OPENNI2);
```
It has different properties which can be accessed via `VideoCapture::get` and `::set` methods: http://docs.opencv.org/3.3.0/dc/dfc/group__videoio__flags__others.html#ga904c71f1f5b910a0ae1048b9e8dd0603